utils/classify_topic.py

```
@@ -0,0 +1,58 @@
+        return "정치"
+    if topic in social_topics:
+        return "사회"
+    if topic == 'conflict, war and peace':
+        return "국제"
+    if topic in culture_topics:
+        return "문화"
+    if topic == 'sport':
+        return "스포츠"
+    if topic in science_topics:
+        return "과학"
+    return none;
```

Ruff (0.11.9)

Undefined name `none`

(F821)